boards/avr-rss2/include/board.h

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2019 Robert Olsson <roolss@kth.se>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_avr-rss2
+ * @{
+ *
+ * @file
+ * @brief       Board definitions for the rss2 256rfr2 board.
+ *
+ * @author      Robert Olsson <roolss@kth.se>
+ *
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "rss2.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name   Baudrate for STDIO terminal
+ *
+ * The standard configuration for STDIO in spu/atmega_comman/periph/uart.c
+ * is to use double speed.
+ *
+ *
+ * Matches this with BAUD in Board/Makefile.include
+ *
+ * @{
+ */
+#ifndef STDIO_UART_BAUDRATE
+#define STDIO_UART_BAUDRATE (115200)       /**< Sets Baudrate for e.g. Shell */
+#endif
+/** @} */
+
+
+/**
+ * @name   LED pin definitions and handlers
+ * @{
+ */
+#define LED_PORT            PORTE
+#define LED_PORT_DDR        DDRE
+
+#define LED0_PIN             (1 << LED_RED )
+#define LED1_PIN             (1 << LED_YELLOW)
+
+#define LED0_MASK           (1 << DDE4)
+#define LED1_MASK           (1 << DDE3)
+
+#define LED0_OFF            (LED_PORT |=  LED0_MASK)
+#define LED0_ON             (LED_PORT &= ~LED0_MASK)
+#define LED0_TOGGLE         (LED_PORT ^=  LED0_MASK)
+
+#define LED1_OFF             (LED_PORT |=  LED1_MASK)
+#define LED1_ON             (LED_PORT &= ~LED1_MASK)
+#define LED1_TOGGLE         (LED_PORT ^=  LED1_MASK)
+
+
+/**
+ * @name    Usage of LED to turn on when a kernel panic occurs.
+ * @{
+ */
+#define LED_PANIC            LED0_ON //LED_RED_ON
+/** @} */
+
+  
+/**
+ * @name DS18 pins  OW_BUS_0
+ * @{
+ */
+#define DS18_PARAM_PIN  0xD7
+#define DS18_PARAM_PULL   (GPIO_IN_PU)
+  
+/**
+ * @name xtimer configuration values
+ * @{
+ */
+#define XTIMER_DEV          TIMER_DEV(0)
+#define XTIMER_CHAN         (0)
+#define XTIMER_WIDTH        (16)
+#define XTIMER_HZ                   (62500UL)
+//#define XTIMER_BACKOFF              (40)
```

Comment:
Just remove commented out code - and keep everything on the same indentation level ;)